packages/components/tsconfig.json

```diff
@@ -1,25 +1,26 @@
 {
   "compilerOptions": {
-    "outDir": "dist/",
-    "module": "commonjs",
-    "target": "es5",
-    "lib": ["es5", "es6", "dom"],
-    "sourceMap": true,
     "allowJs": false,
-    "jsx": "react",
-    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
     "baseUrl": "src",
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
     "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "lib": ["es5", "es6", "dom"],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noImplicitAny": false,
```

Comment:
I know you just alphabetized, but I'd recommend

- Turn on `noImplicitAny`
- Remove all these `noImplicitFoo` config options, and set `strict: true` (which turns them all on)